gnarly.rb

```diff
@@ -393,4 +394,20 @@ def modify_npm_scripts
   end
 end
 
+def format_ruby
+  gsub_file "app/mailers/application_mailer.rb",
+    "default from: 'from@example.com'\nlayout 'mailer'",
+    "default from: \"from@example.com\"\nlayout \"mailer\""
+
+  gsub_file "Gemfile", /\n{3,}/, "\n\n"
+
+  gsub_file "spec/rails_helper",
+    "require File.expand_path('../../config/environment', __FILE__)",
+    "require File.expand_path(\"../config/environment\", __dir__)"
+
+  gsub_file "spec/rails_helper",
+    "Rails.root.join(\"spec/support/**/*.rb\")",
+    "Rails.root.join(\"spec\", \"support\", \"**\", \"*.rb\")"
+end
```

Comment:
Related to #18, did you experiment with adding a call to `bundle exec rubocop --auto-correct` (granted, experimental) to see if that resolves these issues (as well as) presumably others that exist in the code base?

That may be easier/more comprehensive than addressing these issue by issue, though I know that command is limited in its scope itself/not perfect.

Reply:
No but I'll give that a try.

Reply:
Works pretty well, it resolved all but one of these. Should we require usage of `--auto-correct` and then correct the remainders by hand?

Reply:
Yes, I would make this part of the template itself. Doing some cleanup, something like this:

```ruby
def post_bundle
  after_bundle do
    setup_react if react?
    remove_dir "test"
    git :init

    # Moving in here because it's important that this be done after bundle is run so that rubocop is definitely available.
    format_ruby
    completion_notification
  end
end

def completion_notification
  puts ""
  ascii_art
  post_install_instructions
end

def format_ruby
  run "bundle exec rubocop --auto-correct"
  # TODO: fix other issues
end
```

Reply:
I'm pessimistic that this is the case, but after running auto-correct and fixing the other issue you mentioned, is the generated application free of rubocop errors? It may be outside of the scope of this, but to close #18, I'd like to add a step to the CI process (somewhere around [here](https://github.com/TheGnarCo/gnarails/blob/31fa3c5c425caf6336b68cefa70f0779d3c481b5/.circleci/config.yml)) which runs rubocop on the generated application and makes sure it has no errors.

While generally I disagree with rubocop breaking the build for projects, if the intention for this specific code base to generate code that adheres 100% with our style guide, then I'd be in favor of the build breaking if a change was made that did not result in that being the case. 

Otherwise, we could write a script that whitelists very specific errors coming back from rubocop, run that script in CI, and make sure that only those style guide violations occur and no others.

There may be things like config keys in the dev environment that span over 100 lines or something that we don't believe should be modified, and those are the things we should whitelist in the CI script (or modify the rubocop file of the generated application itself to exclude those very specific instances).

Again, that may be a separate item than what you intend to address here, but I'm speaking more holistically in what I'd consider closing #18.